### PR TITLE
Clarification on goals and PRs for firmware

### PR DESCRIPTION
**Pull request by TheButlah**
CC @ImUrX @Noxime @carl-anders @kitlith 

Some links may be broken. I'll fix that eventually

Please note the scope of feedback that I'm primarily interested in:

* Any missing clarifications that are needed?
* Typos, prose
* Broken links

Opinions about the goals as listed are welcome too, although they are unlikely to change.

### CODE REVIEW

File: firmware/docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -22,12 +22,19 @@ work, like `cargo build`, `cargo doc --open`, etc.
   on save in your IDE :)
 * Don't use any dependencies that are GPL or viral or closed-source licenses (ask me if
   you are unsure)
-* If you want, see [tips for making a simple PR](Pull Request Tips.md) to increase the
-  speed that we review PRs.
-* Read the [project goals](Goals.md) if you are interested.
+* See [Pull Request Tips.md] to increase the speed that we review PRs.
+* Read [Goals.md] if you are interested about the project goals.
```
Comment (ImUrX) [inline firmware/docs/CONTRIBUTING.md]: linking doesn't work

File: firmware/README.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 An implementation of SlimeVR firmware, built in Rust. Uses `embedded-hal` instead of
 Arduino.
 
+## Project Goals
+Read [Goals.md](docs/Goals.md)
+
+## Contributing
+Read [CONTRIBUTING.md]
```
Comment (ImUrX) [inline firmware/README.md]: linking doesnt work

**Review by ImUrX — CHANGES_REQUESTED** [review]

**Comment by TheButlah** [comment]
Gave it a second pass to make goals simpler.

File: firmware/docs/CONTRIBUTING.md (hunk 1 above)
Comment (TheButlah) [reply firmware/docs/CONTRIBUTING.md]: fixed

File: firmware/README.md (hunk 2 above)
Comment (TheButlah) [reply firmware/README.md]: Fixed

**Comment by TheButlah** [comment]
will merge tomorrow end of day (GMT-5) if no further clarification is needed

**Review by ImUrX — APPROVED** [review]

**Pull request merged**